control/veda_file_discovery.py

```diff
@@ -32,6 +32,7 @@
 
 logging.basicConfig(level=logging.INFO)
 logging.getLogger("requests").setLevel(logging.WARNING)
+logging.getLogger("boto").setLevel(logging.CRITICAL)
```

Comment:
I think it's enough to do `ERROR` level instead?

Reply:
You're probably right. Changing it